math32/math.go

```diff
@@ -804,6 +804,17 @@ func ClampInt(x, a, b int) int {
 	return x
 }
 
+// Clamp64 clamps x to the provided closed interval [a, b]
+func Clamp64(x, a, b float64) float64 {
```

Comment:
Please just be reasonable and make **one** `math32.Clamp` generic function. All scalar `math32` functions should be generic (moreover, I really think the whole package should be generic and not called `math32`).